frontend/src/pages/analytics/data-management/AnalyticsDataManagement.tsx

```diff
@@ -55,6 +63,43 @@ const AnalyticsDataManagement: React.FC = () => {
     },
   ];
 
+  const onClickTriggerScan = async () => {
+    try {
+      const { success }: ApiResponse<any> = await triggerScan(
+        defaultStr(projectId)
+      );
+      if (success) {
+        alertMsg(
+          t('analytics:metadata.scanTriggeredSuccessfully'),
+          COMMON_ALERT_TYPE.Success as AlertType
```

Comment:
use `success` instead of COMMON_ALERT_TYPE (remove COMMON_ALERT_TYPE if no neccessary)

Reply:
COMMON_ALERT_TYPE  removed